accessibility-checker-engine/test/v2/checker/accessibility/rules/landmark_name_unique_ruleunit/example_3_pass.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<!--
+   /******************************************************************************
+     Copyright:: 2020- IBM, Inc
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+  *****************************************************************************/
+-->
+
+<html lang="en-US">
+
+<head>
+</head>
+
+<body id="body">
+	<!-- PASS FAIL POTENTIAL RECOMENDATION???  
+	Example 3: Two navs not differentiated by name in the same region - This is a problem. Not sure if fail. We would flag this with HTML ruleset and fail on IBM's ruleset. ARIA 11 says this is bad form... But not a failure. This should be a recomendation. 
```

Comment:
This comment doesn't describe the test case below, which has two navigation regions (not navs), and they _are_ differentiated by name. This is a pass. There's no bad form or anything.

Reply:
Old comment that should be removed.